server/libmain/libmain.go

```diff
@@ -342,6 +343,9 @@ func StartAndRunServices(env *real_environment.RealEnv) {
 		log.Fatalf("%v", err)
 	}
 
+	// Register the codec for all RPC servers and clients.
+	vtprotocodec.Register()
```

Comment:
Do you need to call this from executor main as well?

Reply:
Done.